tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -103,10 +105,25 @@ func (c *cluster) bootstrap() error {
 		}
 	}
 
-	c.Stressers = make([]Stresser, len(members))
+	c.Stressers = make([]Stresser, 0)
+	c.LeaseStressers = make([]LeaseStresser, len(members))
+	// init stressers
 	for i, m := range members {
-		c.Stressers[i] = c.stressBuilder(m)
-		go c.Stressers[i].Stress()
+
+		lsStresser := c.leaseStresserBuilder(m, i)
```

Comment:
can we combine leaseStresserBuilder + lsStresser.Setup to newLeaseStresser(c, m, i)? We do not usually use builder pattern in go.